service/lib/dinstaller/dbus/manager.rb

```diff
@@ -65,7 +65,8 @@ def initialize(backend, logger)
         dbus_reader :status, "u"
 
         # Error messages when the status is 0
-        dbus_reader :error_messages, "as"
+        dbus_reader :error_messages, "as",
+          emits_changed_signal: ::DBus::EmitsChangedSignal.new(:invalidates)
```

Comment:
@jreidinger 

```rb
 dbus_reader :error_messages, "as", emits_changed_signal: :invalidates
```

is the intended usage

Reply:
... and your way will `raise` as soon as the file is parsed

Reply:
... actually it will not because a `dbus_reader` *without* an assignment method (`error_messages=`) means no `dbus_watcher` is created and the `e_c_s:` option is never used.